components/admin-tabs/PortalSettings.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { adminTabs } from '../../common/constants';
+import { getWelcome } from '../../common/apiClient';
+import useSWR from 'swr';
+import PortalSettingsRow from './PortalSettingsRow';
+
+const PortalSettings = () => {
+  const { data: welcome } = useSWR('/api/v1/welcome', getWelcome);
+  const dummyData = [
+    { setting: 'Open Date', value: '12/20/21' },
+    { setting: 'Close Date', value: '12/20/21' },
+    { setting: 'Confirm By Date', value: '12/20/21' },
```

Comment:
can you change these to datestrings instead? then also to set them, you can use something like this:

https://ant.design/components/date-picker/#header